tests/scripts/pre-validation-vsi-quickstart.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-./pre-validation-generate-ssh-key.sh ssh_key
+./pre-validation-generate-ssh-key.sh ssh_public_key
```

Comment:
why was this changed? The variable name for quickstart is `ssh_key` so please change it back

Reply:
I have changed the variable name to `ssh_public_key` to make it inline with other patterns, as we are adding new pattern I thought this shouldn’t impact existing user, should I revert it back to `ssh_key`?

Reply:
yes please revert it back - the VSI quickstart DA is already in production, so lets leave it as is